src/main/resources/config/application.properties-sample

```diff
@@ -41,15 +41,23 @@ appium.click.adb = false
 #IN CASE OF WEB
 #parameters for web driver create
 webdriver.browser.name = Chrome
+#in case you specified this param, we will try to automatically find corresponding driver version
+webdriver.browser.version = 63
 webdriver.starting.url = http://google.com
 
+#in case if you want drivers to be downloaded automatically
+webdriver.nexus.url = http://your.nexus.repo.with.drivers.com/
```

Comment:
Are we can using only nexus?

Reply:
No, but if this param is specified, and no webdriver.driver.path is specified, we will download drivers from nexus.

Reply:
I mean can we using taobao, or something else?